resources/lang/pt-br/accounts.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ *    Copyright 2015-2016 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed in the hopes of
+ *    attracting more community contributions to the core ecosystem of osu!
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'edit' => [
+        'title' => 'Configurações de Conta',
+        'title_compact' => 'configurações',
+
+        'avatar' => [
+            'title' => 'Editar Avatar',
```

Comment:
We agreed to use capitalized letters only at the start of new sentences. I think it would be better to use "Editar avatar" instead of "Editar Avatar".